include/z64play.h

```diff
@@ -0,0 +1,161 @@
+#ifndef Z64PLAY_H
+#define Z64PLAY_H
+
+#include "ultra64.h"
+
+#include "prerender.h"
+
+#include "z64actor.h"
+#include "z64bgcheck.h"
+#include "z64camera.h"
+#include "z64cutscene.h"
+#include "z64environment.h"
+#include "z64frame_advance.h"
+#include "z64game.h"
+#include "z64game_over.h"
+#include "z64interface.h"
+#include "z64light.h"
+#include "z64message.h"
+#include "z64object.h"
+#include "z64pause.h"
+#include "z64scene.h"
+#include "z64sfx_source.h"
+#include "z64skybox.h"
+#include "z64sram.h"
+#include "z64transition.h"
+#include "z64view.h"
+
+union Color_RGBA8_u32;
+struct QuestHintCmd;
+struct VisMono;
+
+typedef struct SequenceContext {
+    /* 0x00 */ u8 seqId;
+    /* 0x01 */ u8 natureAmbienceId;
+} SequenceContext; // size = 0x2
```

Comment:
At first I thought this should probably have its own header, but on second thought it really shouldn't. It's holding sequence information to be passed around to other subsystems used by Play.

That said we should probably try to come up with a new name for it at some point. "Context" to me means "separate system". 

I think `SceneSequences` could work. They are the sequence values that come from the scene itself. The variables are also heavily involved with the `Environment_PlaySceneSequence` and `Audio_PlaySceneSequence` functions. 

You could argue further that this could be considered a part of the "scene" system itself. But currently I view it as the Play system storing this info from the scene to then pass to other systems as needed.

Reply:
I like the `SceneSequences` name so I'll rename it, thanks for the suggestion